test/integrations/destinations/hs/router/data.ts

```diff
@@ -100,6 +101,235 @@ export const data = [
       },
     },
   },
+  {
+    name: 'hs',
+    description: 'legacy router retl tests',
```

Comment:
Fix the test cases by breaking duplicating components of the test cases as constants

Reply:
This is legacy one I guess we can remove this test case  rather than investing anytime to break duplicate components

Reply:
This is an old test case with no changes only the order if it got changed due which it is showing the difference